internal/uvm/timezone.go

```diff
@@ -0,0 +1,78 @@
+package uvm
+
+import (
+	"fmt"
+
+	hcsschema "github.com/Microsoft/hcsshim/internal/hcs/schema2"
+	"golang.org/x/sys/windows"
+)
+
+var gmtTimezone = &hcsschema.TimeZoneInformation{
+	Bias:         0,
+	StandardName: "GMT Standard Time",
+	StandardBias: 0,
+	DaylightName: "GMT Daylight Time",
+	DaylightBias: -60,
```

Comment:
You mentioned also finding info on a "UTC" time zone. Did that one have DST? I would rather we use something that doesn't have DST if we can find a good standard.

Reply:
It does not iirc, I'm working to swap it over and rebase this PR right now

Reply:
Done